include/git2/diff.h

```diff
@@ -588,6 +588,9 @@ typedef enum {
 	/** Turn on all finding features. */
 	GIT_DIFF_FIND_ALL = (0x0ff),
 
+	/** Does no work on trying to find renames. */
+	GIT_DIFF_FIND_NO_RENAMES = (1u << 8),
```

Comment:
Wait, I'm confused. We already have a flag `GIT_DIFF_FIND_RENAMES`, why do we need another flag `GIT_DIFF_FIND_NO_RENAMES`? Shouldn't it be sufficient to just unset the bit for `GIT_DIFF_FIND_RENAMES`?

Reply:
The logic for this was to toggle `GIT_DIFF_FIND_ALL` when encountering `NO_RENAMES`. At least, that's what I recall from 2 years ago.

Reply:
I guess it would make more sense to toggle that whenever `GIT_DIFF_FIND_RENAMES` is not set, then?

Reply:
You can use other flags without using `GIT_DIFF_FIND_RENAMES`. The option I  chose was to toggle off all find features with this flag, as the defaults/options may change over time. If you have any other preferred way of handling this, do tell. I find no other way to make a catch-all, disable rename heuristics when I first wrote this patch.